src/main/java/uk/ac/cam/cl/dtg/segue/dao/users/PgUsers.java

```diff
@@ -704,10 +704,16 @@ private RegisteredUser createUser(final RegisteredUser userToCreate) throws Segu
                             + "date_of_birth, gender, registration_date, school_id, "
                             + "school_other, exam_board, last_updated, email_verification_status, "
                             + "last_seen, email_verification_token, email_to_verify, "
-                            + " registered_contexts, registered_context_last_confirmed) "
-                    + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);",
+                            + "registered_contexts, registered_context_last_confirmed) "
+                    + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?::text::jsonb[], ?);",
```

Comment:
Why is this JSONB[] -> TEXT -> JSONB[] cast necessary? JDBC seems to set it as a JSONB[] value on L733, so this looks like it ought to be an unnecessary overhead?

Reply:
Hmm yes you're right. I remember trying it both ways in case it made a difference but must have not moved back. Like you say we explicitly create the array of the correct type so the prepared statement shouldn't need casting.